### PR TITLE
Remove JWT auth method

### PR DESCRIPTION
**Pull request by theSoenke**
phrase-ruby contains an issue triggered by JWT auth which is overwriting headers set by token auth. JWT tokens are also no publicly usable yet

### CODE REVIEW

File: doc/compiled.json [hunk 1]
```diff
@@ -1127,6 +1119,14 @@
...
             }
           ],
+          "teams": [
+            {
+              "id": "04d36d845576b9d494d05e0b70fe813c",
+              "name": "Team 1",
+              "created_at": "2020-12-07T12:56:21Z",
+              "updated_at": "2020-12-07T12:56:21Z"
+            }
+          ],
```
Comment (vermaxik) [inline doc/compiled.json:1129]: doesn't look it relates to issue with JWT, added by accident?

File: doc/compiled.json [hunk 2]
```diff
@@ -1308,7 +1308,38 @@
...
+              }
+            },
+            "example": {
+              "id": "abcd1234cdef1234abcd1234cdef1234",
+              "name": "My Android Project",
+              "project_role": "Developer",
+              "main_format": "xml",
+              "created_at": "2015-01-28T09:52:53Z",
+              "updated_at": "2015-01-28T09:52:53Z"
+            }
```
Comment (vermaxik) [inline doc/compiled.json:1342]: added by accident?

Comment (theSoenke) [reply doc/compiled.json:1342]: the autogenerated output was just not added by a previous PR and I thought it makes sense to keep it consistent

**Review by vermaxik — APPROVED** [review]
👍

**Pull request merged**